### PR TITLE
fix: refine the warning with respect to start --optimzed and the db

### LINKED ISSUE #32348: none of the enabled features are shown as such in the admin console
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

Despite building the server with a list of ```Preview``` features, none of those appear in the ```Enabled features``` area but rather below the ```Disabled features```.
Are they not used or is this just a UI issue?

### Version

25.0.4

### Regression

- [ ] The issue is a regression

### Expected behavior

All enabled features, including ```Preview``` and ```Experimental``` ones should be listed as enabled when keycloak has been built/started with those as enabled.

### Actual behavior

When enabling the following features at build time:
- client-secret-rotation,
- passkeys,
- persistent-user-sessions,
- recovery-codes

I get:
Screenshot from 2024-08-22 15-49-02

### How to Reproduce?

1. download 25.0.4 and extract
2. install ojdbc11-23.5.0.24.07.jar into providers
3. install orai18n-23.5.0.24.07.jar into providers
4. set keycloak.conf and quarkus.properties into conf 
5. bin/kc.sh build --features="client-secret-rotation,passkeys,persistent-user-sessions,recovery-codes"
6. bin/kc.sh start --proxy-headers=xforwarded &

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by shawkins**
closes: #32348

### CODE REVIEW

File: docs/guides/server/db.adoc [hunk 1]
```diff
@@ -137,11 +137,8 @@ This command includes the minimum settings needed to connect to the database.
 
 The default schema is `keycloak`, but you can change it by using the `db-schema` configuration option.
 
-[WARNING]
-Do NOT use the `--optimized` flag for the `start` command if you want to use a particular DB (except the H2).
-Executing the build phase before starting the server instance is necessary.
-You can achieve it either by starting the instance without the `--optimized` flag,
-or by executing the `build` command before the optimized start.
+`start --optimized` will use database type and other build-time values associated with the current state of the optimized image.
```
Comment (mabartos) [inline docs/guides/server/db.adoc:140]:
```suggestion
[WARNING]
Command `start --optimized` will use database type and other build-time values associated with the current state of the optimized build. 
```

1. I'd rather keep the Warning block as it's more visible to users, and we could avoid any other further confusion. WDYT?
2. Perhaps "optimized build" should be preferred over "optimized image" in this context, right?

**Review by mabartos — COMMENTED** [review]
@shawkins Just nitpicks.

**Review by vmuzikar — APPROVED** [review]
LGTM as is, thank you @shawkins.

File: docs/guides/server/db.adoc (hunk 1 above)
Comment (vmuzikar) [reply docs/guides/server/db.adoc:140]: I agree the warning might be a bit misleading in this case.

**Comment by shawkins** [comment]
Moving to a draft - the user who opened the issue will contribute a pr https://github.com/keycloak/keycloak/issues/32348#issuecomment-2315515028 

If for whatever reason that doesn't happen, I'll update this with the approch from the issue which will be a more wholistic rewrite of the section and an elimination of the warning.

**Pull request closed without merging**